api/src/main/java/org/openmrs/scheduler/db/SchedulerDAO.java

```diff
@@ -39,6 +39,13 @@
 	 */
 	public TaskDefinition getTask(Integer taskId) throws DAOException;
 	
+	/**
+	 * @see org.openmrs.scheduler.SchedulerService#getTaskByUuid(String uuid)
+	 * 
+	 * @throws DAOException
```

Comment:
Do you need the above throws clause after having the see annotation.

Reply:
ooh not really,let me remove it